CDCgov/prime-reportstream
# Turn Okta base URL into a variable; same for production, different for test/staging

This PR varies the Okta instance different environments are using.

Test Steps:
1. Run the terraform and ensure the function app configuration value stays as-is

## Changes
- Make `OKTA_baseUrl` a variable; define that variable in each environment we're deploying to

## Checklist

### Testing

### Process
